src/components/safe-apps/SafeAppCard/index.tsx

```diff
@@ -159,7 +159,7 @@ const SafeAppCardListView = ({
 }: SafeAppCardViewProps) => {
   return (
     <SafeAppCardContainer safeAppUrl={safeAppUrl} onClickSafeApp={onClickSafeApp}>
-      <CardContent>
+      <CardContent sx={{ pb: '16px !important' }}>
```

Comment:
The default was 24px and it wasn't symmetrical.

Before:
<img width="366" alt="Screenshot 2023-03-22 at 09 44 51" src="https://user-images.githubusercontent.com/381895/226856923-85840167-59da-4958-a3d6-03a175420f93.png">

After:
<img width="376" alt="Screenshot 2023-03-22 at 09 44 35" src="https://user-images.githubusercontent.com/381895/226856970-6aaa3e56-d1a7-4740-965b-26492e62b9c9.png">